front/nginx.conf

```diff
@@ -19,6 +19,7 @@ server {
         proxy_set_header Host $http_host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+	    proxy_set_header X-Forwarded-Proto $scheme;
```

Comment:
Please use 8 spaces to align this statement with the other statements.

FTR: this is required since [1.7.0](https://github.com/hedgedoc/hedgedoc/releases/tag/1.7.0)

Reply:
Somehow VSCode decided to add 1 tab and 4 spaces instead of 8 spaces...
Did not see that in VSCode itself, so thank you for pointing out!